src/server/game/Spells/Spell.h

```diff
@@ -218,6 +218,8 @@ enum SpellEffectHandleMode
 
 typedef std::list<std::pair<uint32, ObjectGuid>> DispelList;
 
+static const uint32 SPELL_INTERRUPT_NONPLAYER = 32747;
```

Comment:
the number is just in description, it s not in dbc or anywhere else... what should i do with that?